src/dotnet-sourcelink/dotnet-sourcelink.csproj

```diff
@@ -4,6 +4,7 @@
     <TargetFrameworks>$(NetCurrent)</TargetFrameworks>
     <!-- Allow tool to roll forward to a newer major version. -->
     <RollForward>Major</RollForward>
+    <ExcludeFromSourceBuild>true</ExcludeFromSourceBuild>
```

Comment:
Why does this tool need to be excluded from source build? Please add a comment as it isn't clear just from reading the code.

Reply:
It should be included. This is not a shipping product yet but it is meant to be. It's for customers to test that their binaries have correct Source Link.

Reply:
>This is not a shipping product yet

Can this be done as a separate issue to not block this PR? Or it being excluded for some time is a no-go?

Reply:
IMO if source build repositories currently don't depend on the tool, it shouldn't be an issue to exclude it for now and track enabling it.

Reply:
Created an issue for it -> https://github.com/dotnet/sourcelink/issues/1028